modules/preview-react/InformationHighlight/lib/hooks/modelHook.ts

```diff
@@ -1,15 +1,18 @@
 import {createModelHook} from '@workday/canvas-kit-react/common';
 
-export type Variant = 'emphasis' | 'caution' | 'attention';
+export type Variant = 'informational' | 'caution' | 'critical';
+export type Emphasis = 'low' | 'high';
 
 export const useInformationHighlightModel = createModelHook({
   defaultConfig: {
-    variant: 'emphasis' as Variant,
+    variant: 'informational' as Variant,
```

Comment:
I would just inline the string union for cleaner interface and typesense
```suggestion
    variant: 'informational' as 'informational' | 'caution' | 'critical',
```

Reply:
same for emphasis

Reply:
right not i don't have it inlined since variant and emphasis are exported to other components, how would inlining it impact that? Or would we keep the exports

Reply:
just delete the exports and inline it. I think after refactoring everything you shouldn't need those types